clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -622,6 +619,81 @@ public void testUnsupportedTLSVersion() throws Exception {
         server.verifyAuthenticationMetrics(0, 1);
     }
 
+    /**
+     * Tests that connections fails if TLSv1.3 enabled but cipher suite suitable only for TLSv1.2 used.
+     */
+    @Test
+    public void testCiphersSuiteForTLSv1_2_FailsForTLSv1_3() throws Exception {
+        if (!Java.IS_JAVA11_COMPATIBLE)
+            return;
```

Comment:
We can use `assume` to do this in a more idiomatic way. Same for other places where we do something similar.

Reply:
Fixed.